lib/discover.js

```diff
@@ -50,6 +50,18 @@ TesselSeeker.prototype.start = function(opts) {
     var tessel = new Tessel(conn);
     var opening = tessel.connection.open()
       .then(function() {
+        if (tessel.lanConnection && opts.authorized !== undefined) {
```

Comment:
I think what we should do here is just [change the Tessel initialization](https://github.com/tessel/t2-cli/blob/master/lib/tessel/tessel.js#L5) to have `authorized = true` by default if it is a USB Connection. That way this can just be a simple if statement:

```
if (opts.authorized && !tessel.connection.authorized) {
  return;
}
```